### PR TITLE
Update QuickWins.rst

### PR DESCRIPTION
**Pull request by plorence**
added a few bullets about strategies for identifying quick wins

### CODE REVIEW

File: QuickWins.rst [hunk 1]
```diff
@@ -17,3 +17,21 @@ Starting notes:
...
+
+• Triage
+
+Seek out and embrace the triage process; your quick wins ought to be focused on "Severity 1 / Priority 1" issues. "Stopping the bleeding" is a quick win itself! Insist on attending bug triages (or backlog review, or any sort of formal prioritization process or bug scrub); doing so will immerse you quickly into the pain points of the product(s) and help you envision and prioritize first deliverables. (Note that the highest source of pain may be customer-facing or internal. You may not necessarily have a charter to produce internal documentation, but if you are alleviating pain, you could make a case for producing something quickly for an internal audience. See above.)
```
Comment (jlara310) [inline QuickWins.rst]: Suggestion:

"doing so will immerse you quickly into" -> "doing so will quickly immerse you into"

File: QuickWins.rst [hunk 2]
```diff
@@ -17,3 +17,21 @@ Starting notes:
...
+
+* Use a "production values" strategy
+
+Filmmakers use the term "production values" to priortize those things which will end up in the frames of the final cut that viewers will see. For example, the dollars spend on catering to feed the cast and crew -- while important and necessary -- aren't actually *seen* in the end-product by the consumers of the film. Therefore, it may be wiser to spend more money on costumes or makeup rather than sandwiches. If your charter is to prioritize end-user-facing documentation, you may want to be brutually honest about those deliverable that will provide the largest value for your time.
```
Comment (jlara310) [inline QuickWins.rst]: Small typo: "the dollars spend" -> "the dollars spent"

File: QuickWins.rst (hunk 1 above)
Comment (plorence) [reply QuickWins.rst]: sure

File: QuickWins.rst (hunk 2 above)
Comment (plorence) [reply QuickWins.rst]: yep; also "deliverables"

**Comment by jlara310** [comment]
Looks, good. Thanks!

**Pull request merged**